src/main/java/uk/gov/justice/hmpps/prison/service/CaseNoteService.java

```diff
@@ -126,7 +123,7 @@ public CaseNote createCaseNote(final Long bookingId, @NotNull @Valid @CaseNoteTy
             .agencyLocation(booking.getLocation())
             .type(caseNoteTypeReferenceCodeRepository.findById(CaseNoteType.pk(caseNote.getType())).orElseThrow(EntityNotFoundException.withId(caseNote.getType())))
             .subType(caseNoteSubTypeReferenceCodeRepository.findById(CaseNoteSubType.pk(caseNote.getSubType())).orElseThrow(EntityNotFoundException.withId(caseNote.getSubType())))
-            .noteSourceCode(caseNoteSource)
+            .noteSourceCode(caseNote.isSystemGenerated() ? "AUTO" : "INST")
```

Comment:
I see that you've dropped config property `api.caseNote.sourceCode` but it's [set in a couple of places](https://github.com/search?q=repo%3Aministryofjustice%2Fprison-api+api.caseNote.sourceCode&type=code). Best to keep it configurable if not AUTO?